recipes/cloudml-hypertune/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "cloudml-hypertune" %}
+{% set version = "0.1.0.dev6" %}
```

Comment:
As this is a dev version, please upload it to a dev label until there is a release. You will need to add an adjusted version of https://github.com/conda-forge/lief-feedstock/blob/dev/recipe/conda_build_config.yaml here

Reply:
@xhochy Updated

Reply:
Sorry to necro this thread, but thought we made an exception for packages that [have only had dev releases]( https://github.com/conda-forge/cfep/blob/b561fa3d8a45cf0dcd6d9f8085cd42067301ef93/cfep-05.md#exceptions ), which I think this fits under. Though please feel free to correct me if I've misunderstood something